bindings/python/iota_sdk/types/unlock.py

```diff
@@ -84,11 +88,36 @@ class NftUnlock:
     type: int = field(default_factory=lambda: int(UnlockType.Nft), init=False)
 
 
+@json
+@dataclass
+class MultiUnlock:
+    """Unlocks a MultiAddress with a list of other unlocks.
+    """
+    unlocks: List[Unlock]
```

Comment:
doesnt this need 
```
= field(
        metadata=config(
            decoder=deserialize_unlock
        ))
```

?

Reply:
Hmm idk, we don't have it in the unlocks of a payload

Reply:
we have it on most other inheriting types, i think we need it.
Can you add it to the other one as well then?

Reply:
Done